### PR TITLE
Introduce `cname` argument to initSdk call

### PR DESCRIPTION
**Pull request by nemscep**
- Introduce `cname` initSdk method argument
- Pass cname parameter to respective android and ios sdks

### CODE REVIEW

File: example/lib/main.dart [hunk 1]
```diff
@@ -17,15 +17,15 @@ void main() async {
...
+  const String exampleCname = 'YOUR-CNAME-HERE';
 
   try {
     await sdk.initSdk(
-      apiDomain: exampleAppApiDomain,
-      apiKey: exampleAppApiKey,
-    );
+        apiDomain: exampleAppApiDomain,
+        apiKey: exampleAppApiKey,
+        cname: exampleCname);
```
Comment (navaronbracke) [inline example/lib/main.dart:28]:
```suggestion
        cname: exampleCname,
     );
```

File: lib/src/platform_interface/gigya_flutter_plugin_platform_interface.dart [hunk 2]
```diff
@@ -84,6 +84,7 @@ abstract class GigyaFlutterPluginPlatform extends PlatformInterface {
   Future<void> initSdk({
```
Comment (navaronbracke) [inline lib/src/platform_interface/gigya_flutter_plugin_platform_interface.dart:84]: @nemscep Could you update the documentation to mention `[cname]` as well?

File: lib/gigya_flutter_plugin.dart [hunk 3]
```diff
@@ -88,11 +88,13 @@ class GigyaSdk {
   Future<void> initSdk({
```
Comment (navaronbracke) [inline lib/gigya_flutter_plugin.dart:88]: @nemscep Could you update the documentation to mention cname as well?

**Comment by nickwph** [comment]
hello! just checking if this will be merged soon. me and my team has been waiting for this feature and hope it will be released soon!

**Comment by nemscep** [comment]
> hello! just checking if this will be merged soon. me and my team have been waiting for this feature and hope it will be released soon!

Hello! Well I guess it depends on the owners of this repo who have write access 😄

**Pull request merged**